saleor/plugins/webhook/plugin.py

```diff
@@ -915,14 +915,23 @@ def draft_order_deleted(self, order: "Order", previous_value: Any) -> Any:
                 order_data, event_type, webhooks, order, self.requestor
             )
 
-    def fulfillment_created(self, fulfillment: "Fulfillment", previous_value):
+    def fulfillment_created(
```

Comment:
Please also update typing in `base_plugin.py` file:
`fulfillment_created: Callable[["Fulfillment", Any], Any] `
to:
` fulfillment_created: Callable[["Fulfillment", bool, Any], Any]`